src/transforms/groupby.js

```diff
@@ -39,11 +39,11 @@ exports.attributes = {
         valType: 'string',
         description: [
             'Pattern by which grouped traces are named. If only one trace is present,',
-            'defaults to the group name (`"%g"`), otherwise defaults to the group name',
-            'with trace name (`"%g (%t)"`). Available escape sequences are `%g`, which',
-            'inserts the group name, and `%t`, which inserts the trace name. If grouping',
+            'defaults to the group name (`"%{group}"`), otherwise defaults to the group name',
+            'with trace name (`"%{group} (%{trace})"`). Available escape sequences are `%{group}`, which',
```

Comment:
This plural vs singular problem came up too in https://github.com/plotly/plotly.js/pull/1770 :arrow_right:  [here](https://github.com/plotly/plotly.js/blob/b2e54861dfe9960a7bb447f0a622d4198a4e22c7/src/components/fx/helpers.js#L105-L106) - where values of _plural_ attributes are listed using a singular (grammatically) key.

Maybe we should start declaring corresponding _singular_ key in the plot schema and have `Lib.templateString` and `appendArrayPointValue` use it.